app/src/main/java/ca/chronofit/chrono/onboarding/OnBoardActivity.kt

```diff
@@ -0,0 +1,122 @@
+package ca.chronofit.chrono.onboarding
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import androidx.viewpager.widget.ViewPager
+import ca.chronofit.chrono.MainActivity
+import ca.chronofit.chrono.R
+import ca.chronofit.chrono.databinding.ActivityOnboardBinding
+import ca.chronofit.chrono.util.BaseActivity
+
+class OnBoardActivity : BaseActivity() {
+    private lateinit var bind: ActivityOnboardBinding
+    private var welcomeFrag: Fragment = OnBoardWelcomeFrag()
```

Comment:
I would say rename this to onBoardWelcomeFrag, onBoardCircuitFrag because i think we use circuitFrag and swatchFrag else where. remove the confusion.

Reply:
Everything is within the OnboardActivity so no need to add that to each one that would be redundant